src/generator/writers/writeFunction.ts

```diff
@@ -136,6 +136,30 @@ export function writeStatement(f: ASTStatement, self: string | null, returns: Ty
         });
         ctx.append(`}`);
         return;
+    } else if (f.kind === 'statement_try') {
+        ctx.append(`try {`);
+        ctx.inIndent(() => {
+            for (const s of f.statements) {
+                writeStatement(s, self, returns, ctx);
+            }
+        });
+        ctx.append('} catch (_) { }');
```

Comment:
Is it valid FunC? Shouldn't `catch` have two arguments (`catch(_,_)`) in this case?

Reply:
we can do it this way because of FunC's tensor types.
`_` in this case is a pair that consists of both `catch` arguments.

I did it this way because we don't need to access values themselves here.